### ISSUE TITLE
[BUG] No Test ads

### ISSUE THREAD

**Issue by kyoz** [Poing-Studios/godot-admob-ios#5]
**Describe the bug**

Although i have turned `real` off but it seem the plugin only show real ads. I'v check the `settings.json` file and it's correct. is_real=false. Also i export ios with debug.

I'v check xcode output and getting this, which is the real banner id i'v set in the box.

Doesn't know if i'm missing any step. @@

```
021-08-24 10:12:58.574789+0700 testing_app[1392:304981] banner will load with the banner id ca-app-pub-9564990221035664/9221509333
```

<img width="716" alt="Screen Shot 2021-08-24 at 10 16 00" src="https://user-images.githubusercontent.com/29815830/130550178-d744cf74-ea2e-465a-abf6-a00cd6d03ace.png">

**Expected behavior**

Should be able to show test ad for testing purpose

**Comment by gumaciel** [comment]
Hi @kyoz thank you for opening this issue.

Please check if the xcode output says this: `on Testing Simulator:` or `on Testing Real Device: testDeviceIdentifiers`.

If is saying this, then probably then probably there's nothing wrong with the plugin.

Check this:

1. Your `GADApplicationIdentifier` to your AdMob: https://developers.google.com/admob/ios/quick-start#update_your_infoplist
2. Banner, Interstitial, Rewarded or RewardedInterstitial Ads with your Unit IDs of AdMob
3. Mark `Real`
4. Unmark `Test Europe User Consent`
5. Export as `release`

On next days we will do a update that you won't need to remember to mark the `Real` or `Testing Europe User Consent` because the [editor plugin](https://github.com/Poing-Studios/godot-admob-editor) will check if the Game is on `release` or `debug` mode, then will set `Real` to `true` or `false` depending where the user is running the app.

https://github.com/Poing-Studios/godot-admob-editor/issues/6

**Comment by kyoz** [comment]
Thank @gumaciel, it maybe my mistake to forget to toggle Real of first. By toggle it off now i can see:

```
2021-08-24 19:00:33.845255+0700 testing_app[1680:385975] on Testing Real Device: testDeviceIdentifiers: eefcf3e4578b26d876f5a99225223fa6
```

But doesn't know why the banner ad just not showing (That's why i think i'v got blocked by request for real ad in dev environment 😆 

BTW, i'll wait for next release, i'm happy to test it.

**Comment by gumaciel** [comment]
@kyoz Hey, now i see you want to show Test Ads, so you need to set Real to `false`

**Comment by kyoz** [comment]
Ya, but it just not showing, although i've check the `Show Banner Instanly`. In Xcode i can see this:

```
2021-08-24 19:19:18.259221+0700 testing_app[1707:390010] init admob plugin
2021-08-24 19:19:18.259637+0700 testing_app[1707:390010] initialize admob
2021-08-24 19:19:22.023111+0700 testing_app[1707:390010] AdMob Module not initialized
2021-08-24 19:19:22.026860+0700 testing_app[1707:390010] on Testing Real Device: testDeviceIdentifiers: eefcf3e4578b26d87221299225223fa6
2021-08-24 19:19:22.028291+0700 testing_app[1707:390010] maxAdContentRating = GADMaxAdContentRatingGeneral
2021-08-24 19:19:25.411471+0700 testing_app[1707:390010] GADMobileAds : 1
2021-08-24 19:19:25.414891+0700 testing_app[1707:390010] Calling load_banner
2021-08-24 19:19:25.415317+0700 testing_app[1707:390010] banner will load with the banner id ca-app-pub-9564124121035664/9221509317
2021-08-24 19:19:25.416287+0700 testing_app[1707:390010] banner position = 0
2021-08-24 19:19:25.417198+0700 testing_app[1707:390010] Adaptive banner will be created
```

Now i see `AdMob Module not initialized`. Hmm, i'll check my installation again.

**Comment by gumaciel** [comment]
@kyoz This is the function to initialize the AdMob:

https://github.com/Poing-Studios/godot-admob-ios/blob/master/plugin/admob/src/main/admob.mm#L195-L211

Is called on the method: https://github.com/Poing-Studios/godot-admob-ios/blob/master/plugin/admob/src/main/admob.mm#L140

You need to wait to the signal "_on_AdMob_initialization_complete" emits

Are you testing with the `Example.tscn` scene?

**Comment by gumaciel** [comment]
The `MobileAds.initialize()` is calling this function on iOS or Android:

https://github.com/Poing-Studios/godot-admob-editor/blob/master/addons/admob/scripts/MobileAds.gd#L19

But when the `MobileAds` loads, then automatically calls the `initialize()` so you don't need to worry about calling this

If the MobileAds initialize successfully, then this function will be called on the `Example.tscn`: https://github.com/Poing-Studios/godot-admob-editor/blob/master/addons/admob/develop/Example.gd#L59-L75

**Comment by gumaciel** [comment]
But as I can see, on this line is saying the MobileAds was successfully initialized: 

```
2021-08-24 19:19:25.411471+0700 testing_app[1707:390010] GADMobileAds : 1
```
1 = true
https://github.com/Poing-Studios/godot-admob-ios/blob/master/plugin/admob/src/main/admob.mm#L200

So this is not the problem.

**Comment by gumaciel** [comment]
AdMob support says this:


```
Wait up to one hour after you create an app or ad unit.

In some cases, it can take a few days for ads to appear in new apps or ad units. 

Note: New iOS apps will not show Google ads until they’re listed in the Apple App Store. Learn how to test your ads. 

Newly registered apps and newly created ad units typically take up to an hour to activate.

Because of this, you may not see live impressions immediately.
```

**Comment by gumaciel** [comment]
Ps: Try to avoid to post your personal data in the history of this Issue because anyone can try to use they and might cause problems to you.

Like: DeviceIdentifier, Unit IDs.

**Comment by kyoz** [comment]
Thanks @gumaciel , i'll review everything. Cause i use one codebase to test for android & ios so maybe there's some problem in my code. I'll try to review it and let if you if i found anything.

**Comment by gumaciel** [comment]
@kyoz I saw you put a Unit Id on the beginning of the Issue so it can't be deleted, but I can delete the Issue for you if you want to, so all the history will be deleted:

![image](https://user-images.githubusercontent.com/20030153/130620429-b4762853-d753-4fae-93fd-f5e6b72c9541.png)

**Comment by kyoz** [comment]
yes please delete it. I'll open another one if i found out any issue, THank you somuch @gumaciel
